subt/tools/logschecker.py

```diff
@@ -39,12 +34,26 @@ def main(logfile, streams):
     print("-"*60)
     relevant_streams_id = [lookup_stream_id(logfile, name) for name in streams.keys()]
     stat_list = [np.array([])] * len(relevant_streams_id)
+    ecoder_stream_id = lookup_stream_id(logfile, "kloubak.encoders")
+    was_time_in_sec = None
+    time_diff = None
     with LogReader(logfile, only_stream_id=relevant_streams_id) as log:
         for timestamp, stream_id, data in log:
             time_in_sec = timestamp.total_seconds()
             item_id = relevant_streams_id.index(stream_id)
             stat_list[item_id] = np.append(stat_list[item_id], time_in_sec)
 
+            if stream_id == ecoder_stream_id:
+                encoders = deserialize(data)
+                for num in encoders:
```

Comment:
`num` is typically `number` ... here I would use rather `diff` or `difference` or `enc`